file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/run-cli-j-geJYS3.js:325
        throw new Error($, { cause: h });
              ^

Error: [09-Jun-2025 07:29:27 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO wp_aioseo_blc_cache
SET `key` = 'aioseo_blc_activation_redirect', `value` = 'b:1;', `expiration` = '2025-06-09 07:29:57', `created` = '2025-06-09 07:29:27', `updated` = '2025-06-09 07:29:27'
ON DUPLICATE KEY UPDATE `value` = 'b:1;', `expiration` = '2025-06-09 07:29:57', `updated` = '2025-06-09 07:29:27'
/* 1 = 1 */</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN IMMEDIATE | (no parameters)</li>
<li>Executing: INSERT INTO `wp_aioseo_blc_cache` SET `key` = 'aioseo_blc_activation_redirect' , `value` = 'b:1;' , `expiration` = '2025-06-09 07:29:57' , `created` = '2025-06-09 07:29:27' , `updated` = '2025-06-09 07:29:27' ON CONFLICT DO UPDATE SET `value` = 'b:1;' , `expiration` = '2025-06-09 07:29:57' , `updated` = '2025-06-09 07:29:27' | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 near "SET": syntax error
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('INSERT INTO wp_...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('INSERT INTO wp_...')
#3 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('INSERT INTO wp_...')
#4 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Core/Database.php(1156): wpdb->get_results('INSERT INTO wp_...', 'OBJECT')
#5 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Core/Cache.php(150): AIOSEO\BrokenLinkChecker\Core\Database->run()
#6 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Main/Activate.php(69): AIOSEO\BrokenLinkChecker\Core\Cache->update('activation_redi...', 'b:1;', '2025-06-09 07:2...')
#7 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Main/Activate.php(40): AIOSEO\BrokenLinkChecker\Main\Activate->showSetupWizard()
#8 /wordpress/wp-includes/class-wp-hook.php(324): AIOSEO\BrokenLinkChecker\Main\Activate->activate(false)
#9 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#10 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#11 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_broken...', false)
#12 /internal/eval.php(20): activate_plugin('broken-link-che...')
#13 {main}</pre>
 for query INSERT INTO wp_aioseo_blc_cache
SET `key` = 'aioseo_blc_activation_redirect', `value` = 'b:1;', `expiration` = '2025-06-09 07:29:57', `created` = '2025-06-09 07:29:27', `updated` = '2025-06-09 07:29:27'
ON DUPLICATE KEY UPDATE `value` = 'b:1;', `expiration` = '2025-06-09 07:29:57', `updated` = '2025-06-09 07:29:27'
/* 1 = 1 */ made by activate_plugin, do_action('activate_broken-link-checker-seo/aioseo-broken-link-checker.php'), WP_Hook->do_action, WP_Hook->apply_filters, AIOSEO\BrokenLinkChecker\Main\Activate->activate, AIOSEO\BrokenLinkChecker\Main\Activate->showSetupWizard, AIOSEO\BrokenLinkChecker\Core\Cache->update, AIOSEO\BrokenLinkChecker\Core\Database->run, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[09-Jun-2025 07:29:39 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO wp_aioseo_blc_cache
SET `key` = 'aioseo_blc_activation_redirect', `value` = 'b:1;', `expiration` = '2025-06-09 07:30:09', `created` = '2025-06-09 07:29:39', `updated` = '2025-06-09 07:29:39'
ON DUPLICATE KEY UPDATE `value` = 'b:1;', `expiration` = '2025-06-09 07:30:09', `updated` = '2025-06-09 07:29:39'
/* 1 = 1 */</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN IMMEDIATE | (no parameters)</li>
<li>Executing: INSERT INTO `wp_aioseo_blc_cache` SET `key` = 'aioseo_blc_activation_redirect' , `value` = 'b:1;' , `expiration` = '2025-06-09 07:30:09' , `created` = '2025-06-09 07:29:39' , `updated` = '2025-06-09 07:29:39' ON CONFLICT DO UPDATE SET `value` = 'b:1;' , `expiration` = '2025-06-09 07:30:09' , `updated` = '2025-06-09 07:29:39' | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 near "SET": syntax error
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('INSERT INTO wp_...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('INSERT INTO wp_...')
#3 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('INSERT INTO wp_...')
#4 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Core/Database.php(1156): wpdb->get_results('INSERT INTO wp_...', 'OBJECT')
#5 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Core/Cache.php(150): AIOSEO\BrokenLinkChecker\Core\Database->run()
#6 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Main/Activate.php(69): AIOSEO\BrokenLinkChecker\Core\Cache->update('activation_redi...', 'b:1;', '2025-06-09 07:3...')
#7 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Main/Activate.php(40): AIOSEO\BrokenLinkChecker\Main\Activate->showSetupWizard()
#8 /wordpress/wp-includes/class-wp-hook.php(324): AIOSEO\BrokenLinkChecker\Main\Activate->activate(false)
#9 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#10 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#11 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_broken...', false)
#12 /internal/eval.php(20): activate_plugin('broken-link-che...')
#13 {main}</pre>
 for query INSERT INTO wp_aioseo_blc_cache
SET `key` = 'aioseo_blc_activation_redirect', `value` = 'b:1;', `expiration` = '2025-06-09 07:30:09', `created` = '2025-06-09 07:29:39', `updated` = '2025-06-09 07:29:39'
ON DUPLICATE KEY UPDATE `value` = 'b:1;', `expiration` = '2025-06-09 07:30:09', `updated` = '2025-06-09 07:29:39'
/* 1 = 1 */ made by activate_plugin, do_action('activate_broken-link-checker-seo/aioseo-broken-link-checker.php'), WP_Hook->do_action, WP_Hook->apply_filters, AIOSEO\BrokenLinkChecker\Main\Activate->activate, AIOSEO\BrokenLinkChecker\Main\Activate->showSetupWizard, AIOSEO\BrokenLinkChecker\Core\Cache->update, AIOSEO\BrokenLinkChecker\Core\Database->run, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[09-Jun-2025 07:29:51 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO wp_aioseo_blc_cache
SET `key` = 'aioseo_blc_activation_redirect', `value` = 'b:1;', `expiration` = '2025-06-09 07:30:21', `created` = '2025-06-09 07:29:51', `updated` = '2025-06-09 07:29:51'
ON DUPLICATE KEY UPDATE `value` = 'b:1;', `expiration` = '2025-06-09 07:30:21', `updated` = '2025-06-09 07:29:51'
/* 1 = 1 */</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN IMMEDIATE | (no parameters)</li>
<li>Executing: INSERT INTO `wp_aioseo_blc_cache` SET `key` = 'aioseo_blc_activation_redirect' , `value` = 'b:1;' , `expiration` = '2025-06-09 07:30:21' , `created` = '2025-06-09 07:29:51' , `updated` = '2025-06-09 07:29:51' ON CONFLICT DO UPDATE SET `value` = 'b:1;' , `expiration` = '2025-06-09 07:30:21' , `updated` = '2025-06-09 07:29:51' | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 near "SET": syntax error
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('INSERT INTO wp_...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('INSERT INTO wp_...')
#3 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('INSERT INTO wp_...')
#4 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Core/Database.php(1156): wpdb->get_results('INSERT INTO wp_...', 'OBJECT')
#5 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Core/Cache.php(150): AIOSEO\BrokenLinkChecker\Core\Database->run()
#6 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Main/Activate.php(69): AIOSEO\BrokenLinkChecker\Core\Cache->update('activation_redi...', 'b:1;', '2025-06-09 07:3...')
#7 /wordpress/wp-content/plugins/broken-link-checker-seo/app/Main/Activate.php(40): AIOSEO\BrokenLinkChecker\Main\Activate->showSetupWizard()
#8 /wordpress/wp-includes/class-wp-hook.php(324): AIOSEO\BrokenLinkChecker\Main\Activate->activate(false)
#9 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#10 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#11 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_broken...', false)
#12 /internal/eval.php(20): activate_plugin('broken-link-che...')
#13 {main}</pre>
 for query INSERT INTO wp_aioseo_blc_cache
SET `key` = 'aioseo_blc_activation_redirect', `value` = 'b:1;', `expiration` = '2025-06-09 07:30:21', `created` = '2025-06-09 07:29:51', `updated` = '2025-06-09 07:29:51'
ON DUPLICATE KEY UPDATE `value` = 'b:1;', `expiration` = '2025-06-09 07:30:21', `updated` = '2025-06-09 07:29:51'
/* 1 = 1 */ made by activate_plugin, do_action('activate_broken-link-checker-seo/aioseo-broken-link-checker.php'), WP_Hook->do_action, WP_Hook->apply_filters, AIOSEO\BrokenLinkChecker\Main\Activate->activate, AIOSEO\BrokenLinkChecker\Main\Activate->showSetupWizard, AIOSEO\BrokenLinkChecker\Core\Cache->update, AIOSEO\BrokenLinkChecker\Core\Database->run, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[09-Jun-2025 07:30:03 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function BABO\insert_with_markers() in /wordpress/wp-content/plugins/backup-bolt/inc/activation.php:41
Stack trace:
#0 /wordpress/wp-content/plugins/backup-bolt/inc/activation.php(26): BABO\Babo_Activator->babo_backup_htaccess('/wordpress/wp-c...')
#1 /wordpress/wp-content/plugins/backup-bolt/backup-bolt.php(92): BABO\Babo_Activator->__construct()
#2 /wordpress/wp-includes/class-wp-hook.php(324): babo_activate(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_backup...', false)
#6 /internal/eval.php(20): activate_plugin('backup-bolt/bac...')
#7 {main}
  thrown in /wordpress/wp-content/plugins/backup-bolt/inc/activation.php on line 41
[09-Jun-2025 07:30:13 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function BABO\insert_with_markers() in /wordpress/wp-content/plugins/backup-bolt/inc/activation.php:41
Stack trace:
#0 /wordpress/wp-content/plugins/backup-bolt/inc/activation.php(26): BABO\Babo_Activator->babo_backup_htaccess('/wordpress/wp-c...')
#1 /wordpress/wp-content/plugins/backup-bolt/backup-bolt.php(92): BABO\Babo_Activator->__construct()
#2 /wordpress/wp-includes/class-wp-hook.php(324): babo_activate(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_backup...', false)
#6 /internal/eval.php(20): activate_plugin('backup-bolt/bac...')
#7 {main}
  thrown in /wordpress/wp-content/plugins/backup-bolt/inc/activation.php on line 41
[09-Jun-2025 07:30:22 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function BABO\insert_with_markers() in /wordpress/wp-content/plugins/backup-bolt/inc/activation.php:41
Stack trace:
#0 /wordpress/wp-content/plugins/backup-bolt/inc/activation.php(26): BABO\Babo_Activator->babo_backup_htaccess('/wordpress/wp-c...')
#1 /wordpress/wp-content/plugins/backup-bolt/backup-bolt.php(92): BABO\Babo_Activator->__construct()
#2 /wordpress/wp-includes/class-wp-hook.php(324): babo_activate(false)
#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_backup...', false)
#6 /internal/eval.php(20): activate_plugin('backup-bolt/bac...')
#7 {main}
  thrown in /wordpress/wp-content/plugins/backup-bolt/inc/activation.php on line 41
[09-Jun-2025 07:32:27 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>bafg</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[09-Jun-2025 07:32:38 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>bafg</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[09-Jun-2025 07:32:48 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>bafg</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/run-cli-j-geJYS3.js:325:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async re (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/run-cli-j-geJYS3.js:149:10) {
  [cause]: Error: Error when executing the blueprint step #1 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"better-plugin-version-control"},"options":{"activate":true}}) : Could not download "https://downloads.wordpress.org/plugin/better-plugin-version-control.latest-stable.zip".
  				Check if the URL is correct and the server is reachable.
  				If it is reachable, the server might be blocking the request.
  				Check the browser console and network tabs for more information.
  
  				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
  
  				This means the server that hosts your file does not allow requests from other sites
  				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
  				cross-origin file downloads. Learn more about CORS at
  				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
  
  				If your file is on GitHub, load it from "raw.githubusercontent.com".
  				Here's how to do that:
  
  				1. Start with the original GitHub URL of the file. For example:
  				https://github.com/username/repository/blob/branch/filename.
  				2. Replace "github.com" with "raw.githubusercontent.com".
  				3. Remove the "/blob/" part of the URL.
  
  				The resulting URL should look like this:
  				https://raw.githubusercontent.com/username/repository/branch/filename
  
  				Error:
  				Error: Could not download "https://downloads.wordpress.org/plugin/better-plugin-version-control.latest-stable.zip"
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12093:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ys (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12196:3)
      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/run-cli-j-geJYS3.js:315:48)
      at async re (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/run-cli-j-geJYS3.js:149:10) {
    [cause]: Error: Could not download "https://downloads.wordpress.org/plugin/better-plugin-version-control.latest-stable.zip".
    				Check if the URL is correct and the server is reachable.
    				If it is reachable, the server might be blocking the request.
    				Check the browser console and network tabs for more information.
    
    				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
    
    				This means the server that hosts your file does not allow requests from other sites
    				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
    				cross-origin file downloads. Learn more about CORS at
    				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
    
    				If your file is on GitHub, load it from "raw.githubusercontent.com".
    				Here's how to do that:
    
    				1. Start with the original GitHub URL of the file. For example:
    				https://github.com/username/repository/blob/branch/filename.
    				2. Replace "github.com" with "raw.githubusercontent.com".
    				3. Remove the "/blob/" part of the URL.
    
    				The resulting URL should look like this:
    				https://raw.githubusercontent.com/username/repository/branch/filename
    
    				Error:
    				Error: Could not download "https://downloads.wordpress.org/plugin/better-plugin-version-control.latest-stable.zip"
        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1129:13)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)
  }
}

Node.js v23.11.0
